supabase/migrations/20240313073422_init.sql

```diff
@@ -0,0 +1,1027 @@
+
+alter table "public"."permits" add constraint "permits_nonce_key" UNIQUE using index "permits_nonce_key";
+
+alter table "public"."permits" add constraint "permits_partner_id_fkey" FOREIGN KEY (partner_id) REFERENCES partners(id) not valid;
+
+alter table "public"."permits" validate constraint "permits_partner_id_fkey";
+
+alter table "public"."permits" add constraint "permits_signature_key" UNIQUE using index "permits_signature_key";
+
+alter table "public"."permits" add constraint "permits_token_fkey" FOREIGN KEY (token_id) REFERENCES tokens(id) not valid;
+
+alter table "public"."permits" validate constraint "permits_token_fkey";
+
+alter table "public"."permits" add constraint "permits_transaction_key" UNIQUE using index "permits_transaction_key";
+
+alter table "public"."settlements" add constraint "fk_settlements_location" FOREIGN KEY (location_id) REFERENCES locations(id) ON DELETE CASCADE not valid;
+
+alter table "public"."settlements" validate constraint "fk_settlements_location";
+
+alter table "public"."settlements" add constraint "settlements_credit_id_fkey" FOREIGN KEY (credit_id) REFERENCES credits(id) not valid;
+
+alter table "public"."settlements" validate constraint "settlements_credit_id_fkey";
+
+alter table "public"."settlements" add constraint "settlements_debit_id_fkey" FOREIGN KEY (debit_id) REFERENCES debits(id) not valid;
+
+alter table "public"."settlements" validate constraint "settlements_debit_id_fkey";
+
+alter table "public"."settlements" add constraint "settlements_user_id_fkey" FOREIGN KEY (user_id) REFERENCES users(id) not valid;
+
+alter table "public"."settlements" validate constraint "settlements_user_id_fkey";
+
+alter table "public"."tokens" add constraint "tokens_location_id_fkey" FOREIGN KEY (location_id) REFERENCES locations(id) not valid;
+
+alter table "public"."tokens" validate constraint "tokens_location_id_fkey";
+
+alter table "public"."users" add constraint "users_location_id_fkey" FOREIGN KEY (location_id) REFERENCES locations(id) not valid;
+
+alter table "public"."users" validate constraint "users_location_id_fkey";
+
+alter table "public"."users" add constraint "users_wallet_id_fkey" FOREIGN KEY (wallet_id) REFERENCES wallets(id) not valid;
+
+alter table "public"."users" validate constraint "users_wallet_id_fkey";
+
+alter table "public"."wallets" add constraint "new_wallets_wallet_key" UNIQUE using index "new_wallets_wallet_key";
+
+alter table "public"."wallets" add constraint "wallets_address_check" CHECK ((length(address) = 42)) not valid;
+
+alter table "public"."wallets" validate constraint "wallets_address_check";
+
+alter table "public"."wallets" add constraint "wallets_location_id_fkey" FOREIGN KEY (location_id) REFERENCES locations(id) not valid;
+
+alter table "public"."wallets" validate constraint "wallets_location_id_fkey";
+
+set check_function_bodies = off;
+
+CREATE OR REPLACE FUNCTION public.check_null_permit_tx_on_update()
```

Comment:
same here?

Reply:
This one I do not know, it was there when I dumped the db.